### DISCUSSION TITLE
Special community channels

### DISCUSSION THREAD

**Discussion by ReenigneArcher** [Pycord-Development/pycord#2020] [Q&A] [answered]
Is there any way to get the ID of these special channels?

![image](https://user-images.githubusercontent.com/42013603/233199726-8eeec51e-13d2-4cc0-9f99-10bbabd88729.png)

Well, specifically the `#Browse Channels` and `#Customize Community`. I can mention these channels when typing manually, but not sure how to do it through pycord.
![image](https://user-images.githubusercontent.com/42013603/233199996-47d6c06c-fdfc-4429-bf50-9a21cd73629a.png)

**Comment by plun1331** [comment, marked as answer]
They aren't actual channels, but can be mentioned with the following:
Guide - `<id:home>`
Browse Channels - `<id:browse>`
Customize Community - `<id:customize>`
